docs/advanced_usage.rst

```diff
@@ -277,4 +277,37 @@ also make a switchable widget. You can use it like this:
 
         switched_widget.register('Action type 1', default=True, widget=Widget())
 
-Then instead of switching views, it switches widgets.
\ No newline at end of file
+Then instead of switching views, it switches widgets.
+
+IFrame support is views
+-----------------------------
+
+If some html page has embedded iframes, those can be covered using regular view.
+You just need to set FRAME property for it. FRAME should point out to appropriate iframe and can be xpath and whatever supported by widgetastic.
```

Comment:
I think this should be more specific - can it be a webelement? any kind of locator text?

Reply:
of course, it takes everything what is accepted by browser.element()